README.md

```diff
@@ -492,7 +492,7 @@ class Country < ActiveFile::Base
 
   class << self
     def extension
-      ".super_secret"
+      "super_secret"
```

Comment:
Adding a dot at the first part of `extension` leads our users to seeing some kind of the following error.
```
No such file or directory @ rb_sysopen - /foo/bar/some_file..super_secret (Errno::ENOENT)
```

So removing the dot from the example above may be a little kinder for our users, I think.